filer/static/filer/js/base.js

```diff
@@ -67,28 +67,39 @@ Cl.mediator = new Mediator();
                     $(this).closest(containerSelector).addClass('is-focused');
                 });
 
-                $(containerSelector).find('.dropdown-container').on('show.bs.dropdown', function () {
+                $(containerSelector).find('.filer-dropdown-container').on('show.bs.filer-dropdown', function () {
                     $(containerSelector).addClass('is-focused');
-                }).on('hide.bs.dropdown', function () {
+                }).on('hide.bs.filer-dropdown', function () {
                     $(containerSelector).removeClass('is-focused');
                 });
             }
         }());
 
         // show counter if file is selected
         (function () {
-            var table = $('.navigator-table').find('tr');
+            var navigatorTable = $('.navigator-table').find('tr');
             var actionList = $('.actions-wrapper');
-            var actionSelect = $('.action-select, #action-toggle, .actions .clear a');
+            var actionSelect = $('.action-select');
+
+            // timeout is needed to wait until table row has class selected.
+            setTimeout(function () {
+                if (navigatorTable.hasClass('selected')) {
+                    actionList.addClass('action-selected');
+                }
+            }, 100);
 
-            actionSelect.on('change click', function () {
-                actionList.toggleClass('action-selected', table.hasClass('selected'));
+            actionSelect.on('click', function () {
```

Comment:
I have to use click because change doesn't work. checkbox doesn't change to checked